src/index.html

```diff
@@ -11,7 +11,111 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form_group">
+        <legend>Personal information</legend>
+        <label class="form_field">
+          Surname:
+          <input type="text" name="Surname" 
+            autocomplete="off" required
+          >
+        </label>
+        <label class="form_field">
+          Name:
+          <input type="text" name="Name" 
+            autocomplete="off" required
+          >
+        </label>
+        <label class="form_field">
```

Comment:
```suggestion
        </label>
        
        <label class="form_field">
```
add blank lines between same level multiline elements